org.kde.Kid3.yaml

```diff
@@ -0,0 +1,78 @@
+app-id: org.kde.Kid3
+runtime: org.kde.Platform
+runtime-version: "5.14"
+sdk: org.kde.Sdk
+command: kid3
+
+rename-desktop-file: org.kde.kid3.desktop
+rename-appdata-file: org.kde.kid3.appdata.xml
+rename-icon: kid3-qt
+
```

Comment:
It would be nice to have these renames done upstream instead

Reply:
I went the other way. the application calls itself org.kde.kid3 so better to have that the standard, and not my imposing a capital K.